packages/back-end/src/events/handlers/webhooks/EventWebHookNotifier.ts

```diff
@@ -131,10 +131,6 @@ export class EventWebHookNotifier implements Notifier {
           return { content: data.text };
         }
 
-        case "ms-teams":
-          // TODO: not implemented
-          return eventPayload;
-
```

Comment:
@romain-growthbook Can you share the details of the ms-teams integration - were we planning to eventually build it out, but now we'll just support it via the `raw` type?